README.md

```diff
@@ -1,6 +1,6 @@
 # Fabric Server-side Mods
 
-If you're interested in developing server side Fabric mods, check out the [Fabric Server-side Development](https://discord.gg/PEg2ueH) discord, and submit a Pull Request if you find a server side Fabric mod not listed here.
+If you're interested in developing server side Fabric mods, check out the [Fabric Server-side Development](https://discord.gg/PEg2ueH) Discord, and submit a Pull Request if you find a server side Fabric mod not listed here.
```

Comment:
This seems like an incredibly small change to make a PR for.... but if you're going to do that, you might as well make it grammatically correct.
```suggestion
If you're interested in developing server side Fabric mods, check out the [Fabric Server-side Development](https://discord.gg/PEg2ueH) Discord server, and submit a Pull Request if you find a server side Fabric mod not listed here.
```